src/coreclr/src/vm/bundle.cpp

```diff
@@ -69,7 +69,7 @@ BundleFileLocation Bundle::Probe(LPCWSTR path, bool pathIsBundleRelative) const
         if (_wcsnicmp(m_basePath, path, baseLen) == 0)
 #endif // TARGET_UNIX
         {
-            path += baseLen; // m_basePath includes count for DIRECTORY_SEPARATOR_CHAR_W
+            path += (COUNT_T)baseLen; // m_basePath includes count for DIRECTORY_SEPARATOR_CHAR_W
```

Comment:
Is this just appending to the path now? Or am I missing something here?

Reply:
This is advancing the pointer to skip the path prefix.

Reply:
Won't it use the += overload for `SString` that appends?

Reply:
Oh, just saw that this is outdated now - nevermind.

Reply:
Yup, it was an interleaving commit, I was trying out a cross-platform fix for extended character case that Jan mentioned https://github.com/dotnet/runtime/pull/44466#discussion_r521469700. So far, I have settled on saving the m_basePathString from ctor and using it here (provided all CI installer legs are green).

> Won't it use the += overload for SString that appends?

It is a bit tricky, since the rvalue was `COUNT_T`, the iterator's `operator+=` kicks in: https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/coreclr/src/inc/iterator.h#L111

Reply:
It seems that this would move just the iterator and not change the underlying SString. And even if it does, it would seem as an unexpected side effect.